src/functions/certGen.ts

```diff
@@ -35,11 +35,17 @@ const certGen: Handler = async (
     );
   const certificateUploadService: CertificateUploadService =
     Injector.resolve<CertificateUploadService>(CertificateUploadService);
-  const certificateUploadPromises: Array<Promise<ManagedUpload.SendData>> = [];
+  const certificateUploadPromises: Array<
+    Promise<ManagedUpload.SendData> | Promise<any>
```

Comment:
```suggestion
    Promise<ManagedUpload.SendData> | Promise<ManagedUpload.DeleteData>
```
If that's a thing ?

Reply:
Who knows, dare you to find anything on AWS documentation.

Reply:
In a serious answer, it's this `PromiseResult<S3.DeleteObjectOutput, AWSError>` but I just cannot make it play ball, and for the sake of some better typing on something that is awful anyway, what's the point in sinking hours into this.

Reply:
I got angry and found a solution, what do you think of that change?